src/uwtools/config/core.py

```diff
@@ -325,11 +326,11 @@ def parse_include(self, ref_dict: Optional[dict] = None):
         for key, value in copy.deepcopy(ref_dict).items():
             if isinstance(value, dict):
                 self.parse_include(ref_dict[key])
-            elif isinstance(value, str) and "!INCLUDE" in value:
-                filepaths = value.lstrip("!INCLUDE [").rstrip("]").split(",")
-                # Update the dictionary with the values in the included file.
-                self.update_values(self._load_paths(filepaths))
-                del ref_dict[key]
+            elif isinstance(value, str):
+                if m := re.match(r"^\s*%s\s+(.*)" % INCLUDE_TAG, value):
+                    filepaths = yaml.safe_load(m[1])
+                    self.update_values(self._load_paths(filepaths))
+                    del ref_dict[key]
```

Comment:
Use `re` and `yaml` to process include tags in config files. The behavior of the regex aligns better with how a native YAML tag (which is what is being simulated in non-YAML configs) would behave. For example, the previous version expected exactly one space between the tag and the start of the value, while an actual YAML file would permit an arbitrary number of spaces here -- as the regex does. We can also use `yaml` to parse the actual array-of-strings value.